src/wm_tts_service_EN.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
```

Comment:
Everything depends on the topic one, we should let it there until everythig depends on the service